SharedClasses/ConfigManager.cs

```diff
@@ -38,6 +38,13 @@ public enum Setting
             // Vehicle Chameleon Colours
             vmenu_using_chameleon_colours,
 
+            // Vehicle Prevent Extras When Damaged
+            vmenu_prevent_extras_when_damaged,
+            vmenu_prevent_extras_engine_damage,
+            vmenu_prevent_extras_body_damage,
+            vmenu_prevent_extras_notification_enabled,
+            vmenu_prevent_extras_notification_text,
```

Comment:
While this is a good idea to allow non-English messages, adding localization in a piecemeal fashion like this isn't ideal. Could you remove this please, and hopefully we can properly address it in a future localization update.